### PR TITLE
Split inventory host file

### PR DESCRIPTION
**Pull request by RedOrNot**
per discussion - the inventory host file is now split up per inventory type
+added 1 more check to inventory -> host -> groups and host -> groups verify : make sure all the groups are shown in list mode

### CODE REVIEW

File: cypress/support/hostsfunctions.ts [hunk 1]
```diff
@@ -147,6 +148,7 @@ export function checkHostGroup(host_type: string, organization: Organization) {
       cy.clickModalButton('Confirm');
       cy.contains('button', 'Close').click();
```
Comment (vishsanghishetty) [inline cypress/support/hostsfunctions.ts:149]: please user `cy.getByDataCy` command for this
same for line 150 and 151

**Review by vishsanghishetty — COMMENTED** [review]
you could also create a folder named inventories and move these specs into it

File: cypress/support/hostsfunctions.ts (hunk 1 above)
Comment (RedOrNot) [reply cypress/support/hostsfunctions.ts:149]: thanks 
will do
thanks!

File: cypress/e2e/awx/resources/inventoryHostRegular.cy.ts [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    // use createAndEditAndDeleteHost function in order to test inventory hosts basic functions
+    // after navigating to the right url
+    // the test covers create, verify, edit and delete of hosts form inventory
+    cy.visit(`/infrastructure/inventories/inventory/${inventory.id}/details`);
+    cy.clickTab(/^Hosts$/, true);
+    createAndEditAndDeleteHost('inventory_host', inventory);
+  });
+
+  it('can edit and delete inventory host action from details view', () => {
+    //can and delete host from details view
```
Comment (MilanPospisil) [inline cypress/e2e/awx/resources/inventoryHostRegular.cy.ts:47]: Missing implementation? If not, please skip this test.

**Comment by MilanPospisil** [comment]
> you could also create a folder named inventories and move these specs into it

I agree, this should be inside its own folder. All three files and I would suggest you may move the hostsFunctions file there as well, so its in the same folder, since it is local to those files and those commands will not be shared accross all tests.

File: cypress/e2e/awx/resources/inventoryHostRegular.cy.ts [hunk 3]
```diff
@@ -0,0 +1,95 @@
...
+
+  it.skip("can view a host's facts on the facts tab of a host inside an inventory", () => {
+    //1) Use the inventory, access the host tab of that inventory
+    //2) Visit the host details page, then navigate to the host Facts tab
+    //3) Assert that no facts are showing
+    //4) Create a JT designed to populate the facts for the host, launch the job template
+    //5) Navigate back to the facts tab of that host and assert that the facts exist
+  });
+
+  it.skip('can launch a job template that uses an inventory with a particular host and view the job on the host jobs tab inside the inventory', () => {
```
Comment (MilanPospisil) [inline cypress/e2e/awx/resources/inventoryHostRegular.cy.ts:66]: This test should be part of all three inventory types.

**Comment by MilanPospisil** [comment]
Thanks about the more intuitive organization of tests! Howerver, I noticed that there are some tests missing from original file. Do you want to add the all Sarahs tests scaffoldings here, or do you want to do it in next PR? Since you are not deleting original file in this PR, I suppose this is still work in progress PR?

**Comment by RedOrNot** [comment]
Hi @MilanPospisil  
about this:
https://github.com/ansible/ansible-ui/pull/2423#issuecomment-2145061719
which tests?

I've marge few tests to 1 (groups, action on hosts - create, verify, edit, delete)
did I missed something?

File: cypress/support/hostsfunctions.ts (hunk 1 above)
Comment (RedOrNot) [reply cypress/support/hostsfunctions.ts:149]: that's not working for me at the moment 
why is that change needed?

File: cypress/e2e/awx/resources/inventoryHostRegular.cy.ts (hunk 2 above)
Comment (RedOrNot) [reply cypress/e2e/awx/resources/inventoryHostRegular.cy.ts]: done

File: cypress/e2e/awx/resources/inventoryHostRegular.cy.ts [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    //1) Use inventory and host
+    //2) create a job template that uses that inventory, launch the job template, wait for job to finish
+    //3) Navigate back to inventory -> host tab -> jobs tab -> assert presence of job in that list
+  });
+
+  it.skip('can cancel a currently running job from the host jobs tab inside an inventory', () => {
```
[the remark this answers is not in the document]
Comment (RedOrNot) [reply cypress/e2e/awx/resources/inventoryHostRegular.cy.ts]: done

File: cypress/e2e/awx/resources/inventoryHost/inventoryHostRegular.cy.ts [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+
+  it.skip("can view a host's facts on the facts tab of a host inside an inventory", () => {
+    //1) Use the inventory, access the host tab of that inventory
+    //2) Visit the host details page, then navigate to the host Facts tab
+    //3) Assert that no facts are showing
+    //4) Create a JT designed to populate the facts for the host, launch the job template
+    //5) Navigate back to the facts tab of that host and assert that the facts exist
+  });
+
+  it.skip('can launch a job template that uses an inventory with a particular host and view the job on the host jobs tab inside the inventory', () => {
```
Comment (MilanPospisil) [inline cypress/e2e/awx/resources/inventoryHost/inventoryHostRegular.cy.ts:66]: This test should be in other inv types.

File: cypress/e2e/awx/resources/inventoryHost/inventoryHostRegular.cy.ts [hunk 6]
```diff
@@ -0,0 +1,95 @@
...
+    //1) Use inventory and host
+    //2) create a job template that uses that inventory, launch the job template, wait for job to finish
+    //3) Navigate back to inventory -> host tab -> jobs tab -> assert presence of job in that list
+  });
+
+  it.skip('can cancel a currently running job from the host jobs tab inside an inventory', () => {
```
Comment (MilanPospisil) [inline cypress/e2e/awx/resources/inventoryHost/inventoryHostRegular.cy.ts:72]: This test should be in other inv types.

File: cypress/support/hostsfunctions.ts (hunk 1 above)
Comment (MilanPospisil) [reply cypress/support/hostsfunctions.ts:149]: We will fix this later, sorry, I need to merge this PR so I can also work on that ASAP.

**Review by MilanPospisil — APPROVED** [review]
LGTM :+1:

**Pull request merged**